client/src/App.tsx

```diff
@@ -17,6 +17,9 @@ import { Team } from "./pages/Team";
 import { createAppTheme } from "./styles/theme";
 import { client } from "./graphql/client";
 
+import { Login } from "./pages/Login";
+import { Create } from"./pages/create/Create"
```

Comment:
you don't need create here
it should be done within admin routes - but leave it here for now I'll fix it.